rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -1176,7 +1177,13 @@ def test_upgrade_v12_v13(globaldb: GlobalDBHandler, messages_aggregator):
         assert cursor.execute("SELECT COUNT(*) FROM assets WHERE identifier IN ('HODLSOL','TRISIG')").fetchone()[0] == 0  # noqa: E501
         # token count reduced by 2 due to removal duplicate HODLSOL and TRISIG duplicate entries
         assert cursor.execute('SELECT COUNT(*) FROM assets').fetchone()[0] == tokens_before - 2
+        # solana_tokens count reduced by 5: 2 duplicates removed + 3 user-added tokens migrated to type 'w'  # noqa: E501
+        assert cursor.execute('SELECT COUNT(*) FROM solana_tokens').fetchone()[0] == solana_tokens - 5  # noqa: E501
```

Comment:
the line after this one is affected by the comment above about `reduced by 2 removal of duplicates` so maybe put this one after to have a better understanding